vulnerabilities/csp/source/low.php

```diff
@@ -1,10 +1,11 @@
 <?php
 
-$headerCSP = "Content-Security-Policy: script-src 'self' https://pastebin.com  example.com code.jquery.com https://ssl.google-analytics.com ;"; // allows js from self, pastebin.com, jquery and google analytics.
+$headerCSP = "Content-Security-Policy: script-src 'self' https://pastebin.com http://hastebin.com example.com code.jquery.com https://ssl.google-analytics.com ;"; // allows js from self, pastebin.com, hastebin.com, jquery and google analytics.
```

Comment:
Can you strip the protocol off hastebin, it could end up causing problems.

Reply:
Yes, stripped off in 1c2c645 and tested.